package/MDAnalysis/core/selection.py

```diff
@@ -1056,6 +1056,28 @@ def _apply(self, group):
         return group[np.isin(nmidx, matches)]
 
 
+class LipidSelection(Selection):
+    token = 'lipid'
+
```

Comment:
add a comment from where you got the residue names — you explained it to me during the hackathon and that was very useful information, so capture it here in a comment!

Reply:
@orbeckst I'd be happy to continue working on this PR. I have currently added the lipid names for the charmm 36 ff from the list of lipids in charmm top_all36_lipid.rtf . The selection needs to be extended to other force fields too.

Reply:
I'd like some help with the tests, happy to discuss more in the developer channel on discord.

Reply:
I'll add some more general comments on testing in the PR below. 

Add your sources as a comment in the code, along the lines of
```python
# Lipid residue names from CHARMM's top_all36_lipid.rtf.
# NOTE: need to add names from other FFs.
```